src/FSharpLint.Client/Contracts.fsi

```diff
@@ -0,0 +1,28 @@
+module FSharpLint.Client.Contracts
+
+open System.Threading
+open System.Threading.Tasks
+
+module Methods =
+
+    [<Literal>]
+    val Version: string = "fsharplint/version"
+
+type VersionRequest =
+    {
+        FilePath: string
```

Comment:
@MrLuje the consistency-chaser in me is wanting to suggest that we create here a similar type to Folder, e.g. `File` which uses FileInfo underneath (instead of DirectoryInfo)

Reply:
I'm all for consistency but IMHO there is less value for this one :
* VersionRequest is one of the API entrypoint (whereas `Folder` was only used internally) and it may be less intuitive to use
* File check is already covered by `ErrFileNotFound`, so we may want to change it if we go with the `FileInfo` way

Reply:
I know it has less value, but I'm just allergic to the PrimitiveObsession anti-pattern (since I came to know the type `FileInfo`, seeing a `string` for file paths makes my eyes bleed haha). I can make the change myself, I'll push to your branch yeah?

Reply:
FTR https://hackernoon.com/what-is-primitive-obsession-and-how-can-we-fix-it-wh2f33ki ;)

Reply:
@knocte sure, go for it :)

Reply:
@MrLuje sure, I'll do it this week. BTW can you rebase the PR? looks like there is some conflict

Reply:
@knocte done